specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2025-01-01/examples/ReplicationProtectionClusters_ApplyRecoveryPoint.json

```diff
@@ -0,0 +1,178 @@
+          ],
+          "provisioningState": "Created",
+          "areAllClusterNodesRegistered": true,
+          "clusterRegisteredNodes": [
+            {
+              "clusterNodeFqdn": "sdgql0",
+              "machineId": "3794026b-6792-4d12-9c0c-de0c79376c90",
+              "biosId": "37D0059C-9118-4220-AA1D-58A10EFA7660",
+              "isSharedDiskVirtualNode": false
+            },
+            {
+              "clusterNodeFqdn": "aeeca6a3-171b-4c9d-ae22-0e4adb6416a0",
+              "machineId": "aeeca6a3-171b-4c9d-ae22-0e4adb6416a0",
+              "biosId": "aeeca6a3-171b-4c9d-ae22-0e4adb6416a0",
+              "isSharedDiskVirtualNode": true
+            },
+            {
+              "clusterNodeFqdn": "sdgql1",
+              "machineId": "cb27913a-a5f2-4691-9eba-78b67f45a57a",
+              "biosId": "83CCE932-67EC-4C13-AB29-ACF5F8F7ED48",
+              "isSharedDiskVirtualNode": false
+            }
+          ],
+          "providerSpecificDetails": {
+            "instanceType": "A2A",
+            "multiVmGroupId": "a7ef77cb-ae59-545f-a32a-bf30575ab1c6",
+            "multiVmGroupName": "testcluster",
+            "multiVmGroupCreateOption": "UserSpecified",
+            "primaryFabricLocation": "eastus",
+            "recoveryFabricLocation": "westus",
+            "failoverRecoveryPointId": "/Subscriptions/7c943c1b-5122-4097-90c8-861411bdd574/resourceGroups/resourceGroupPS1/providers/Microsoft.RecoveryServices/vaults/vault1/replicationFabrics/fabric-pri-eastus/replicationProtectionContainers/pri-cloud-eastus/replicationProtectionClusters/testcluster/recoveryPoints/cc48b7f3-b267-432b-ad76-45528974dc62",
+            "clusterManagementId": "a24d47b2-a80b-4553-9c16-499c30c3be07",
+            "rpoInSeconds": 3213,
+            "lastRpoCalculatedTime": "2024-01-22T10:19:11.0490433Z",
+            "initialPrimaryZone": "",
+            "initialPrimaryFabricLocation": "eastus",
+            "initialRecoveryZone": "",
+            "initialRecoveryFabricLocation": "westus",
+            "initialPrimaryExtendedLocation": null,
+            "initialRecoveryExtendedLocation": null,
+            "primaryAvailabilityZone": null,
+            "recoveryAvailabilityZone": null,
+            "primaryExtendedLocation": null,
+            "recoveryExtendedLocation": null,
+            "lifecycleId": "3d523ab9-8c6d-40af-850d-ccee06513dc6"
+          },
+          "sharedDiskProperties": {
+            "protectionState": "UnplannedFailoverCommitRequired",
+            "testFailoverState": "None",
+            "activeLocation": null,
+            "allowedOperations": [],
+            "replicationHealth": "Normal",
+            "healthErrors": [],
+            "currentScenario": null,
+            "sharedDiskProviderSpecificDetails": {
+              "instanceType": "A2A",
+              "managementId": "a24d47b2-a80b-4553-9c16-499c30c3be07",
+              "unprotectedDisks": null,
+              "protectedManagedDisks": [
```

Comment:
>protectedManagedDisks [](http://example.com/codeflow?start=15&length=21)

yuu should update also tests wih the added properties

Reply:
Added.